interface/forms/procedure_order/new.php

```diff
@@ -226,13 +231,27 @@ function getListOptions($list_id , $fieldnames=array('option_id', 'title', 'seq'
  padding-right:2px;
 }
 
+.form-inline
```

Comment:
@robertdown Could you help me in that you are referring to the new.php as a view file? I know we are working toward a separation of concerns. And in this, I was just trying to get it done so there were some obvious shortcuts I took to get the desired effect. My question is so that I am thinking what you are thinking.